### ISSUE TITLE
How to run RL algorithm in OpenAI Gym

### ISSUE THREAD

**Issue by KimSangYeon-DGU** [mlpack/mlpack#1533]
Hi, I want to run the reinforcement learning algorithm that I made using mlpack in OpenAI Gym for test

However, The OpenAI Gym only provides only python version.

So, I have searched the solution to run RL algorithm in OpenAI Gym, and I found that they provides HTTP API for other languages to run.

But there are few information on how to build the program. Although It seems that the program has many errors in running C++ version, the issue page is rather inactive.

So, if anyone knows how to run mlpack algorithm in OpenAI gym, please let us or me know.

Thank you :)

**Comment by HelioStrike** [comment]
I found [this](https://github.com/zoq/gym_tcp_api#getting-started) a few days ago while going through [this page](https://github.com/mlpack/mlpack/wiki/SummerOfCodeIdeas). I haven't used it yet, so I can't give you much info, but I hope it helps. :)

**Comment by KimSangYeon-DGU** [comment]
@HelioStrike 
Thank you so much, I should try it, and then I will share results of method!!

**Comment by KimSangYeon-DGU** [comment]
@HelioStrike 
Hi, HelioStrike!! 
I went through some trial and error to run the program, and in the end, I succeeded to run it correctly.
So, I write the trial and error that I went through
1. `mix deps.get `
To install required dependencies, I should type the command. But,  the error message **Unknown package poolboy in lockfile** is printed.
SOL) rm "~/.hex/cache.ets" 

2. ```
terminate called after throwing an instance of 'boost::system::system_error'
  what():  Connection refused
Aborted (core dumped)
```
SOL) I changed the line 17 code const std::string host = "127.0.0.1"; to const std::string host = "127.0.0.1"; in example.cpp and then the program is working correctly.
